benchmarks/sycl_exclusive_scan.cpp

```diff
@@ -41,25 +41,27 @@ using namespace sycl::helpers;
 /** benchmark_inclusive_scan
  * @brief Body Function that executes the SYCL CG of exclusive_scan
  */
-benchmark<>::time_units_t benchmark_exclusive_scan(const unsigned numReps,
-                                         const unsigned num_elems) {
+benchmark<>::time_units_t benchmark_exclusive_scan(
+    const unsigned numReps, const unsigned num_elems,
+    const cli_device_selector cds) {
   std::vector<int> v1;
 
   for (int i = num_elems; i > 0; i--) {
     v1.push_back(i);
   }
 
   auto exclusive_scan = [&]() {
-    cl::sycl::queue q;
+    cl::sycl::queue q(cds);
     sycl::sycl_execution_policy<class ExclusiveScanAlgorithm1> snp(q);
-    std::experimental::parallel::exclusive_scan(snp, begin(v1), end(v1), 
-      begin(v1), 0, [=](int x, int y){ return x + y; });
+    std::experimental::parallel::exclusive_scan(
+        snp, begin(v1), end(v1), begin(v1), 0,
+        [=](int x, int y) { return x + y; });
```

Comment:
I think it probably is - I ran clang-format over all the files that I'd changed before committing.